src/gluonts/trainer/_base.py

```diff
@@ -102,9 +102,9 @@ class Trainer:
     def __init__(
         self,
         ctx: Optional[mx.Context] = None,
-        epochs: int = 1,
+        epochs: int = 100,
```

Comment:
Finally, 1 didn’t make much sense I think :-)

Reply:
This will make the example on the website super slow.

Reply:
Maybe let’s put an explicit lower number there?

Reply:
Or we could put 10 here instead of 100? Any power of 10 except 1 works for me

Reply:
I would also prefer having an explicit smaller number in the example. We cant really get model that work well out-of-the box otherwise (also 5000 batch update is actually not that slow).

Reply:
But `tutorial.md`seem to set it already, is it the example you are talking about Jan?

Reply:
I think he was referring to the interactive example in the homepage, which probably runs on a 386